config/webpack.config.base.js

```diff
@@ -62,7 +63,7 @@ const MEDIA_URL_LOADER = {
 };
 
 const CSS_LOADER = {
-  loader: 'style!css',
+  loader: ExtractTextPlugin.extract('css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]'),
```

Comment:
in general, I don't recommend copying and pasting code from a tutorial unless you understand exactly what each piece is doing.

> This will add the crazy generated text to the end of the class name
> `<div class="app__element___1MmQg">...</div>`

this pattern is called ["CSS Modules"](https://github.com/css-modules/css-modules), and I believe it requires some extra setup/config to get it working properly. are you planning on employing the CSS Modules pattern? do you need this level of complexity, where each class name gets transformed to become in the form of `[name]__[local]___[hash:base64:5]`? if not, I recommend taking this out.

can you test if this works?

```
loader: ExtractTextPlugin.extract({
  loader: 'css'
  fallbackLoader: 'style',
})
```